mitodl/hugo-course-publisher
# Hide instructor expander while course info section is collapsed

#### Pre-Flight checklist


#### What are the relevant tickets?
Fixes #437 

#### What's this PR do?
 - Hides the expander for the instructor section if the course info section is collapsed
 - Use proper pluralization for "instructors"

#### How should this be manually tested?
Go to a course with 5 or more instructors and verify that the instructors expander button is hidden while the course info section is collapsed

#### Screenshots (if appropriate)
![Screenshot from 2020-12-09 09-12-14](https://user-images.githubusercontent.com/863262/101640558-a6550c00-39fe-11eb-99fd-718b09e0d0c6.png)
